cmd/inspect/inspect_policy_test.go

```diff
@@ -157,3 +168,11 @@ func TestSourcesAndPolicyCantBeBothProvided(t *testing.T) {
 
 	assert.Contains(t, buffy.String(), "Usage:")
```

Comment:
So we need to either remove this check (probably makes the most sense), or assert the inverse. This is because the command chain is being properly constructed, and the root command specifies: `SilenceUsage: true`